controllers/secret_controller.go

```diff
@@ -383,6 +383,9 @@ func createPagerdutyRoute(namespaceList []string) *alertmanager.Route {
 
 		// https://issues.redhat.com/browse/OSD-14071
 		{Receiver: receiverNull, Match: map[string]string{"alertname": "MultipleDefaultStorageClasses", "namespace": "openshift-cluster-storage-operator"}},
+
+		// https://issues.redhat.com/browse/OSD-14857
+		{Receiver: receiverNull, Match: map[string]string{"alertname": "NodeFilesystemAlmostOutOfSpace", "severity": "critical"}},
```

Comment:
Can we just use `MatchRE` to silence the alerts with a label matching the mountpoints we want to ignore? That way we don't have to create a whole new alert,

Reply:
@cblecker - Nice suggestion! Thank you. Yup, I have added `MatchRE` to match the problematic mountpoint to be ignored. I'll close the MCC PR then for new alert.